Romeslayer/potSpotFE
# Feature/react google

#### What's this PR do?

- Converts the application to use newer version of react-google-maps/api

#### Where should the reviewer start?

- App.js and Maps.js

#### How should this be manually tested?

- in CLI run `npm i && npm start`
- Should see a display of map with markers and infowindows when clicked

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)

#### Questions:
